src/ed_object_models/conversion_sdf.py

```diff
@@ -91,6 +118,21 @@ def read_pose(yaml_dict: Mapping) -> str:
     return " ".join(map(str, pose))
 
 
+def create_heightmap(model_name: str, image_path: str, resolution: float, height: float, x: float, y: float) -> str:
+    mesh_path = f"{path.splitext(image_path)[0]}.stl"
+    try:
+        check_call(
+            f"rosrun ed ed_heightmap_to_mesh {image_path} {mesh_path} {resolution} {height} {x} {y}",
+            executable="/bin/bash",
+            shell=True,
```

Comment:
Is there no better way of doing this? Where do we want to keep all the ed model tooling? in ed or ed_object models?

Reply:
The `ed_object_models` pkg is only for TUe. This tool is useful for other people too.